src/components/Main/Schedule/TableCell.tsx

```diff
@@ -16,6 +16,9 @@ export const DataCell = ({
   // We need to keep and update the state of the cell normally
```

Comment:
i merged latest `develop` into my branch, but not sure why some of this logic is showing as a diff